docs/conf.py

```diff
@@ -40,7 +40,9 @@
     # 'recommonmark',
     #'sphinx.ext.napoleon',
     "matplotlib.sphinxext.plot_directive",
-    "nbsphinx"]
+    "sphinxcontrib.bibtex",
+    "nbsphinx",
+    "nbsphinx_link"]
```

Comment:
Is this used? It seems that it needs some additional spec https://nbsphinx-link.readthedocs.io/en/latest/introduction.html

Reply:
@knaaptime